frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-runs.feature

```diff
@@ -350,8 +351,26 @@ Feature: Pipeline Runs
         Scenario: Pipeline Run with parameters: P-07-TC35
             Given pipeline run is displayed for "pipeline-run-parameters" with parameters
               And user is at pipelines page
-            When user clicks Last Run value of "pipeline-run-parameters"
-            Then user will be redirected to Pipeline Run Details page
+             When user clicks Last Run value of "pipeline-run-parameters"
+             Then user will be redirected to Pipeline Run Details page
               And user navigates to pipelineRun parameters tab
               And user is able to see parameters of pipelineRun
               And user is able to see name "testName" and value "testValue" parameters value of pipelineRun
+
+        @regression @odc-4793 @to-do
```

Comment:
This test should be @manual for now. As it requires a 1.8 Red Hat OpenShift Pipelines operator.